server/routerlicious/packages/lambdas-driver/src/document-router/contextManager.ts

```diff
@@ -60,9 +61,10 @@ export class DocumentContextManager extends EventEmitter {
 		context.addListener("checkpoint", (restartOnCheckpointFailure?: boolean) =>
 			this.updateCheckpoint(restartOnCheckpointFailure),
 		);
-		context.addListener("error", (error, errorData: IContextErrorData) =>
-			this.emit("error", error, errorData),
-		);
+		context.addListener("error", (error, errorData: IContextErrorData) => {
+			Lumberjack.verbose("Emitting error from contextManager, context error event.");
```

Comment:
We are not logging any additional data in these changes, just the extra logs saying where the error is being sent from?

Reply:
Yeah just want to track where the error event gets lost or unhandled, because as per the listeners it shouldnt be. The actual error is already logged in the markAsCorrupt method in documentPartition.ts